src/solver/opamSolver.ml

```diff
@@ -90,7 +90,14 @@ let cudf_versions_map universe packages =
     pmap OpamPackage.Map.empty
 
 let name_to_cudf name =
-  Dose_common.CudfAdd.encode (OpamPackage.Name.to_string name)
+  let name_s = OpamPackage.Name.to_string name in
+  if String.for_all (function
```

Comment:
String.for_all is only available since OCaml 4.13. We probably need to introduce `OpamStd.String.for_all`